aderyn_nodejs/create_report.js

```diff
@@ -0,0 +1,12 @@
+const aderyn_driver = require(".");
+
+aderyn_driver.drive("../tests/contract-playground", "report_jsbindings.json", false, [], [])
+
+aderyn_driver.drive_with(
+    "../tests/contract-playground", 
+    "report_jsbindings_config.json",
+    false,
+    [],
+    ["Counter.sol", "Contract.sol"],
+    ["push-zero-opcode","useless-public-function"],
+);
```

Comment:
Basically CI will check that this file re-creates the same assets as normal aderyn cli would